modules/gapi/include/opencv2/gapi/infer/ov.hpp

```diff
@@ -678,8 +678,28 @@ class Params<cv::gapi::Generic> {
     detail::ParamDesc m_desc;
 };
 
+/**
+ * @brief Ask G-API OpenVINO backend to run only inference of model provided.
+ *
+ * G-API OpenVINO backend will perform only the inference of the model provided
+ * without populating input and copying back output data.
+ * This mode is used to evaluate the pure inference performance of the model without
+ * taking into account the i/o data transfer.
+ */
+struct inference_only { };
```

Comment:
1) I'd move it to `cv::gapi::wip`
2) I'd rename it to `benchmark_mode` to illustrate the purpose.

Reply:
Should it be `cv::gapi::wip` or `cv::gapi::wip::ov`?

Reply:
the latter, of course.

Reply:
Done